app/dashboard/templates/profiles/status_box.html

```diff
@@ -61,6 +63,28 @@
       </button>
     </div>
   </div>
+  <div id="attach-dropdown" class="attach text-left mt-4" style="display: none">
+      <div class="row">
+          <div class="col-md-8">
+              <div class="form-group">
+                <label class="sup featured-text">Amount</label>
+                <input id="attachAmount" type="number" class="form-control">
+
+              </div>
+          </div>
+          <div class="col-md-4">
+            <div class="form-group">
+              <label class="sup featured-text">Token</label>
+              <select class="form-control" id="attachToken">
+                <option value="0x0">ETH</option>
+              {% for token in TOKENS %}
+                <option value="{{ token.token_addres }}">{{ token.token_name }}</option>
```

Comment:
Was this only tested with ETH? There is a typo here for the token_address

Reply:
Thanks, I missed that. I tested with some ERC20 tokens like MANA and GNT. 
![image](https://user-images.githubusercontent.com/660973/81989924-b9883080-9603-11ea-8c5f-8797a2828992.png)